.github/workflows/K8s-challange-test.yml

```diff
@@ -0,0 +1,10 @@
+name: k8s-challange-test
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches:
+      - master
```

Comment:
Can you instead of create a new file, extend/edit https://github.com/OWASP/wrongsecrets/blob/master/.github/workflows/minikube-k8s-test.yml to run a new cypress suite only with the tests you currently wrote?